core/src/main/java/org/neo4j/ogm/metadata/ClassInfo.java

```diff
@@ -649,15 +644,16 @@ public Method getMethod(MethodInfo methodInfo) {
      * @return {@link List} of {@link FieldInfo}
      */
     public List<FieldInfo> findIterableFields() {
-        List<FieldInfo> fieldInfos = new ArrayList<>();
+
+        List<FieldInfo> iterableFields = new ArrayList<>();
         try {
             for (FieldInfo fieldInfo : fieldsInfo().fields()) {
                 Class type = getField(fieldInfo).getType();
                 if (type.isArray() || Iterable.class.isAssignableFrom(type)) {
-                    fieldInfos.add(fieldInfo);
+                    iterableFields.add(fieldInfo);
```

Comment:
Just regarding code style: Why is there no streaming api usage as e.g. in the change above?